docs/rfc/rfc-1.adoc

```diff
@@ -435,13 +435,49 @@ As time passes and operators drop out of the system, a wallet becomes at risk
 of being able to meet the 51-of-100 threshold to produce signatures.
 Additionally, we want to avoid situations where operators are the custodians of
 a wallet for extended periods. To avoid these issues, once a wallet is older
-than the `wallet_max_age`, or if it drops below the liveness threshold (say, below 70 on
-a <<heartbeat,heartbeat>>), we motion to transfer the funds to another randomly
-selected wallet.
+than the `wallet_max_age`, or if it drops below the liveness threshold (say,
+below 70 on a <<heartbeat,heartbeat>>), we motion to
+<<closing-a-wallet,transfer the funds>> to another randomly selected wallet.
 
 Once a wallet no longer has funds and is not the primary wallet for new
-deposits, it can be closed and operators are no longer required to maintain
-it.
+deposits, it can be <<closing-a-wallet,closed>> and operators are no longer
+required to maintain it.
+
+[[closing-a-wallet]]
+==== Closing A Wallet
+Governable Parameters:
+
+- `wallet_min_btc`: The smallest amount of BTC a wallet can hold before we
+  attempt to close the wallet and transfer the funds to a randomly selected
+  wallet.
```

Comment:
> Maybe we will only perform that check after redemptions? That would make a bit more sense, but still somebody could make a deposit on a fresh wallet and make a redemption soon afterwards.

Yeah - that's the intention! The definition is weird out of context in a list, so I'll add some clarity.

As for the second point, active wallets can't be closed, and redemptions are still supposed to be coming from the oldest wallets (generally).

-----

Say that `wallet_min_btc = 1 BTC`. If a 3-week old wallet (no longer active) has 1.5 BTC left, and someone skips the dApp to redeem against that wallet instead of the oldest (and we allow that in our validation contract), then it would fall below the threshold and should close. I think this is okay! Closing that wallet is better than making 100 people look after a 0.5 BTC wallet until all of the older wallets get around to finally clearing out.

Reply:
clarified in [0ea9675](https://github.com/keep-network/tbtc-v2/pull/127/commits/0ea96758d96f51a03c7df587d3485964b61e2e5f)